apps/src/javalab/javalabRedux.js

```diff
@@ -40,15 +42,20 @@ export const setSource = (filename, source, isVisible = true) => ({
   isVisible
 });
 
+// Updates the user preferences to reflect change
+export const setIsDarkMode = isDarkMode => {
+  new UserPreferences().setUsingDarkMode(isDarkMode);
+  return {
+    isDarkMode: isDarkMode,
+    type: SET_IS_DARK_MODE
```

Comment:
NIT (and up to you if you want to fix): We haven't been following this convention so far in this file, but actions should generally be named as events rather than setters. [Details from the Redux Style Guide](https://redux.js.org/style-guide/style-guide#model-actions-as-events-not-setters). A name such as `THEME_CHANGED` or `THEME_PREFERENCE_CHANGED` or maybe `COLOR_PREFERENCE_UPDATED` would indicate more clearly what's happening here i.e. an event happened and therefore redux should update its state. Using a name such as "theme" will also streamline adding new themes in the future (i.e. high contrast).

Reply:
Ahh I see. I'll take another look at this. Thank you!